src/LoRa.cpp

```diff
@@ -50,6 +50,8 @@
 
 #define MAX_PKT_LENGTH           255
 
+static SPIClass& SPIPORT = SPI;
```

Comment:
Why not a member variable?

Reply:
Agreed, in order to make this work with multiple SX1276 devices at the same time, this would need to be part of the class.

Reply:
@morganrallen For multiple devices are not multiple spi bus required. So 1 spi will work with 4-7 devices.
![pic](https://user-images.githubusercontent.com/7379702/27146610-c2af92a0-50ee-11e7-8d2a-17bc3722cf01.png)

Reply:
Oh right, I even commented that on #104

Reply:
The important part is the interrupt of each device @morganrallen  are u working on this now?

Reply:
I'm not actively working on this yet, I hope to get time tomorrow or Saturday